Mixpanel/MixpanelInstance.swift

```diff
@@ -40,6 +40,9 @@ open class MixpanelInstance: CustomDebugStringConvertible, FlushDelegate {
 
     /// distinctId string that uniquely identifies the current user.
     open var distinctId = ""
+    
```

Comment:
It looks to me like this is fixed and these violations are outdated. Maybe I'm missing something.